### ISSUE TITLE
[BUG] Build 7057b7c incompatible with LTP (Doom Liquid Textures Pack)

### ISSUE THREAD

**Issue by CandiceJoy** [coelckers/gzdoom#1609]
### GZDoom version

7057b7c

### Which game are you running with GZDoom?

Doom 2

### What Operating System are you using?

Mac OS

### If Other OS, please describe

M1, not Intel

### Relevant hardware info

2021 M1 Pro Macbook Pro 14" 1tb

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

When I run GZDoom 7057b7c with LTP (tried version 5 something and the latest, 6.7), it hangs on loading using 100% CPU.  This seems to happen for me under all circumstances in which LTP is loaded.  Vulkan is also broken in this version; it may or may not be related, I don't know.  

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Install GZDoom 7057b7c
2. Run GZDoom with LTP anywhere in the -file section
3. Watch as your CPU hits 100% :P

### Your configuration

```ini
https://www.toptal.com/developers/hastebin/ivevokoxen.makefile
```

### Provide a Log

GL_VENDOR: Apple
GL_RENDERER: Apple M1 Pro
GL_VERSION: 4.1 Metal - 76.3 (Core profile)
GL_SHADING_LANGUAGE_VERSION: 4.10
GL_EXTENSIONS: GL_ARB_blend_func_extended GL_ARB_draw_buffers_blend GL_ARB_draw_indirect GL_ARB_ES2_compatibility GL_ARB_explicit_attrib_location GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader5 GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_occlusion_query2 GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_subroutine GL_ARB_shading_language_include GL_ARB_tessellation_shader GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_texture_storage GL_ARB_texture_swizzle GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_vertex_attrib_64bit GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_EXT_debug_label GL_EXT_debug_marker GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_sRGB_decode GL_APPLE_client_storage GL_APPLE_container_object_shareable GL_APPLE_flush_render GL_APPLE_rgb_422 GL_APPLE_row_bytes GL_APPLE_texture_range GL_NV_texture_barrier
Max. texture size: 16384
Max. texture units: 16
Max. varying: 16
Max. uniform block size: 65536
Uniform block alignment: 256
Resolution: 3200 x 1800

**Comment by coelckers** [comment]
Please provide a link to this mod.

**Comment by CandiceJoy** [comment]
https://www.moddb.com/addons/doom-liquid-texture-pack

**Comment by coelckers** [comment]
fixed in 1d9da40f5b1d89df0a88afbb1b95df55a5f3ef29
